test/runner.py

```diff
@@ -2,8 +2,6 @@
 import os
 import sys
 
-sys.path.insert(0, os.path.dirname(os.path.dirname(__file__)))
```

Comment:
This line was there to add the directory in which the `test` directory is located to the Python search path (so `python -m test.runner` works), this has nothing to do with the `vsc` namespace at all (which is in `lib/vsc`), so it only concerns the tests.

The vsc-base installation you're testing is controlled via `$PYTHONPATH`